docs/source/developer_guides/quantization.md

```diff
@@ -137,6 +138,27 @@ QLoRA adds trainable weights to all the linear layers in the transformer archite
 config = LoraConfig(target_modules="all-linear", ...)
 ```
 
+## AQLM quantizaion
```

Comment:
Let's add a note that this method is only for LoRA and does not support merging of LoRA adapter weights into the base weights.